conda_build/post.py

```diff
@@ -848,6 +849,44 @@ def library_nature(
 ]
 
 
+def _expand_dsolist(dsolist):
+    result = []
+    for d in dsolist:
+        if d.startswith("*"):
+            result.append(d)
+        else:
+            result.extend([f"**/{basename(f)}" for f in utils.glob(d)])
+    return result
+
+
+def _get_dsolists_prefix(prefix, subdir):
+    allow = []
+    deny = []
+    for fname in utils.glob(join(prefix, "etc", "conda-build", "dsolists.d", "*.json")):
+        with open(fname) as f:
+            contents = json.load(f)
+            if "version" not in contents:
+                raise ValueError("dsolist JSON format requires a version attribute.")
+            version = contents["version"]
+            if version != 1:
+                raise ValueError(f"dsolist version={version} not recognized.")
+            if contents["subdir"] != subdir:
+                continue
+            allow.extend(contents["allow"])
+            deny.extend(contents["deny"])
+    return {"allow": _expand_dsolist(allow), "deny": _expand_dsolist(deny)}
+
+
+def _get_dsolists(build_prefix, host_prefix, subdir):
```

Comment:
This function is Windows only. Should it error out if called for a different target platform?